### ISSUE TITLE
Exporting notes

### ISSUE THREAD

**Issue by lavaplanetnixmirsun** [standardnotes/forum#1294]
Hello,

I've been using Standard Notes for some time (I subscribed to Extended) and I've been testing the export functionality. It's important to me that if Standard Notes is discontinued, I still have access to my notes.

I found that while I can retrieve each note in a .txt file, all notes are in one big folder. I lose all tags associated with them. The notes in the trash are also in that folder.

I think it would be a lot better if there were multiples folders in the exported zip, with the folders corresponding to the tags. The notes could be classified according to their first tag.

I think a trash folder would also be a good idea.

Is anything like this on the road map?

**Comment by markcellus** [comment]
Yeah I agree. Honestly there should be a universal standard for this. If not, any approach would still require some manual work when importing notes into another service.

**Comment by JaspalSuri** [comment]
Hi everyone, thanks for the feedback! These changes currently aren't on the roadmap, but I'll take note of them. 🙂

**Comment by BrunoBernardino** [comment]
Hi @markcellus / @lavaplanetnixmirsun (and anyone else potentially interested in this feature), I've needed this to make my backups more portable, and built a simple local-only CLI tool. Feel free to check it out at https://github.com/BrunoBernardino/standardnotes-folder-export-cli#standard-notes-folder-export-cli---deno as it'll take a unencrypted backup/export and create the files with the tags as directories (standard portable format), and optionally as `.md` files instead of `.txt` too.

**Comment by atmoio** [comment]
@BrunoBernardino nice tool! We have plans to make local backups more robust, now that local file backups have paved the way with for better communication between the web and desktop app.

Feel free to also submit your tool [here](https://github.com/jonhadfield/awesome-standard-notes) 👍

**Comment by BrunoBernardino** [comment]
Thanks @moughxyz ! I've added a PR there.
